build/deploy

```diff
@@ -3,6 +3,7 @@
 git checkout master && \
```

Comment:
This script could be improved by adding this just below the shebang line:

```
set -e
```

This will make it so that any non-zero return code causes the script to cease execution, which means you can remove all the `&& \`.